app/views/candidate_interface/personal_statement/subject_knowledge/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :title, page_title('personal_statement') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @subject_knowledge_form, url: candidate_interface_subject_knowledge_update_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+          <h1 class="govuk-fieldset__heading">
+            <%= t('page_titles.subject_knowledge') %>
```

Comment:
In the prototype, the page heading is `What do you know about the subject you want to teach?`. Think we should match it?